### DISCUSSION TITLE
Is "additional tags" undocumented in the API?

### DISCUSSION THREAD

**Discussion by chucker** [mastodon/mastodon#21635] [Q&A] [answered]
The advanced web client offers an "additional tags" feature. I can subscribe to one tag, then configure it to narrow it down further with "any of these", "all of these", "none of these".

This causes the web app to do a `/api/v1/timelines/tag/PrimaryTag?any[]=MyAnyTag&none[]=MyNoneTag` call. But this syntax — the `any`, etc. query params doesn't _seem_ to be listed over at https://docs.joinmastodon.org/methods/timelines/#tag.

**Comment by trwnh** [comment, marked as answer]
Good catch! Added to https://github.com/mastodon/documentation/pull/1060 (and in the future, reporting such issues on mastodon/documentation would make them easier to see)
